x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/common/import_export/export_prebuilt_rules.ts

```diff
@@ -0,0 +1,502 @@
+
+      const parsedExportResult = parseNdJson(exportResult);
+
+      expect(parsedExportResult).toEqual(
+        expect.arrayContaining([
+          expect.objectContaining({
+            ...PREBUILT_RULE_A['security-rule'],
+            rule_id: PREBUILT_RULE_ID_A,
+          }),
+          expect.objectContaining({
+            ...PREBUILT_RULE_B['security-rule'],
+            rule_id: PREBUILT_RULE_ID_B,
+          }),
+        ])
+      );
+    });
+
+    it('exports customized prebuilt rules via the _export API', async () => {
+      await securitySolutionApi
+        .patchRule({
+          body: {
+            rule_id: PREBUILT_RULE_ID_A,
+            tags: ['custom-tag-a'],
+          },
+        })
+        .expect(200);
+      await securitySolutionApi
+        .patchRule({
+          body: {
+            rule_id: PREBUILT_RULE_ID_B,
+            tags: ['custom-tag-b'],
+          },
+        })
+        .expect(200);
+
+      const { body: secondExportResult } = await securitySolutionApi
+        .exportRules({ query: {}, body: null })
+        .expect(200)
+        .parse(binaryToString);
+
+      expect(parseNdJson(secondExportResult)).toEqual(
+        expect.arrayContaining([
+          expect.objectContaining({
+            rule_id: PREBUILT_RULE_ID_A,
+            immutable: true,
+            rule_source: {
+              type: 'external',
+              is_customized: true,
+            },
+          }),
+          expect.objectContaining({
+            rule_id: PREBUILT_RULE_ID_B,
+            immutable: true,
+            rule_source: {
+              type: 'external',
+              is_customized: true,
+            },
+          }),
+        ])
+      );
+    });
+
+    it('exports custom and prebuilt rules via the _export API', async () => {
+      await Promise.all([
+        securitySolutionApi
+          .createRule({ body: getCustomQueryRuleParams({ rule_id: 'rule-id-1' }) })
+          .expect(200),
+        securitySolutionApi
+          .createRule({ body: getCustomQueryRuleParams({ rule_id: 'rule-id-2' }) })
+          .expect(200),
+      ]);
+
+      const { body: exportResult } = await securitySolutionApi
+        .exportRules({ query: {}, body: null })
+        .expect(200)
+        .parse(binaryToString);
+
+      const exportJson = parseNdJson(exportResult);
+      expect(exportJson).toHaveLength(5); // 2 prebuilt rules + 2 custom rules + 1 stats object
+
+      expect(exportJson).toEqual(
+        expect.arrayContaining([
+          expect.objectContaining({
+            rule_id: PREBUILT_RULE_ID_A,
+            immutable: true,
+            rule_source: {
+              type: 'external',
+              is_customized: false,
+            },
+          }),
+          expect.objectContaining({
+            rule_id: PREBUILT_RULE_ID_B,
+            immutable: true,
+            rule_source: {
+              type: 'external',
+              is_customized: false,
+            },
+          }),
```

Comment:
There's already one non-customized rule in the array, let's add one customized here and remove the `exports customized prebuilt rules via the _export API` scenario altogether.